packages/reactive_ble_mobile/ios/Classes/ReactiveBle/Central.swift

```diff
@@ -167,6 +167,14 @@ final class Central {
             .values
             .forEach(centralManager.cancelPeripheralConnection)
     }
+    
+    func retrievePeripheralName(
+        for peripheralID: PeripheralID
+    ) throws -> String? {
+        let peripheral = try resolve(known: peripheralID)
+        
+        return peripheral.name
+    }
```

Comment:
I doubt the value of this addition. Doesn't look much better than `try c.resolve(known: peripheralID)?.name`.

You're not suggesting to create separate methods here for every property of `CBPeripheral` that we may need to access, are you?

Reply:
Well, it seems the original design-philosophy here was not to expose CoreBluetooth types directly, which justifies your addition (`resolve(known:)` and co. are private).

If you want to stream the updates, on of the `CB-` delegates should be involved.

Reply:
So for this iteration I would suggest not to add the streaming capability yet. 

If the structure of the addition is ok, then we can probably agree on this approach for now?